### PR TITLE
Small fixes to React guide

### PR DESCRIPTION
**Pull request by gaearon**
Misc.

### CODE REVIEW

File: docs/intro-react.md [hunk 1]
```diff
@@ -143,18 +143,21 @@ export default function Cat() {
 }
 ```
 
-Any JavaScript will work between curly braces, including functions like `{fullName("Rum", Tum", "Tugger")}`:
+Any JavaScript expression will work between curly braces, including function calls like `{getFullName("Rum", Tum", "Tugger")}`:
```
Comment (gaearon) [inline docs/intro-react.md]: The convention for function that return something is `get*`

File: docs/intro-react.md [hunk 2]
```diff
@@ -143,18 +143,21 @@ export default function Cat() {
...
 ```SnackPlayer name=Curly%20Braces
 import React from 'react';
 import { Text } from 'react-native';
 
 export default function Cat() {
-  function fullName(firstName, secondName, thirdName) {
+  function getFullName(firstName, secondName, thirdName) {
     return firstName + " " + secondName + " " + thirdName;
   }
+
```
Comment (gaearon) [inline docs/intro-react.md]: Some spacing between function definitions and other code.

File: docs/intro-react.md [hunk 3]
```diff
@@ -354,7 +357,7 @@ import React, {useState} from 'react';
 
 Then you declare the component’s state by calling `useState` inside its function. In this example, `useState` creates an `isHungry` state variable:
 
-```JS
+```jsx
```
Comment (gaearon) [inline docs/intro-react.md]: I think this should fix highlighting

File: docs/intro-react.md (hunk 1 above)
Comment (rachelnabors) [reply docs/intro-react.md]: I almost feel like that would make a great

> note.

Comment (gaearon) [reply docs/intro-react.md]: This is more of a JS convention — can't really teach every one of them here! The reason for it is that unless you look at the definition, `fullName` actually appears to be a name itself. `getFullName` is more clearly a function.

**Pull request merged**